build.gradle

```diff
@@ -75,7 +75,7 @@ buildscript {
     ext {
         // as we don't have 3.0.0, 2.4.0 version for K-NN on darwin we need to keep OpenSearch version as 2.3 for now.
```

Comment:
Can we delete these 2 lines now?

Reply:
This is an automated PR, and will go only in 2.4 branch for now. I will create another PR for removal of this line which will be merged in main.